tools/changelog/changelog.main.kts

```diff
@@ -260,22 +260,31 @@ fun checkPr() {
 
     when {
         releaseNotes is ReleaseNotes.Specified && releaseNotes.entries.isEmpty() -> {
-            err.println("\"## Release Notes\" doesn't contain any items, or section isn't specified")
-            err.println()
-            err.println("See the format in $prFormatLink")
+            err.println("""
+                "## Release Notes" doesn't contain any items, or section isn't specified
+            
+                See the format in $prFormatLink
+            """.trimIndent())
             exitProcess(1)
         }
         releaseNotes is ReleaseNotes.Specified && nonstandardSections.isNotEmpty() -> {
-            err.println("\"## Release Notes\" contains nonstandard sections:")
-            err.println(nonstandardSections.joinToString(", "))
-            err.println()
-            err.println("See possible sections in $prFormatLink#possible-sections")
+            err.println("""
+                "## Release Notes" contains nonstandard section - subsection pairs:
```

Comment:
I'd suggesting putting something around "section - subsection" (like single/double quotes), to clarify the structure of the sentence.

Reply:
Done